Storybook: Add documentation to PluginSignatureBadge story (#51645)
grafana/grafana: packages/grafana-ui/src/components/PluginSignatureBadge/PluginSignatureBadge.mdx

import { Meta, Preview, Props } from '@storybook/addon-docs/blocks';
import { PluginSignatureBadge } from './PluginSignatureBadge';

<Meta title="MDX|PluginSignatureBadge" component={PluginSignatureBadge} />

# Plugin Signature Badge

> The Plugin Signature Badge can be found in each information plugin card of the configuration plugins pannel (Grafana menu > Configuration > Plugins).

Reviewer: ```suggestion
The Plugin Signature Badge can be found in each information plugin card of the configuration plugins panel (Grafana menu > Configuration > Plugins).
```

This is a security measure to make sure plugins haven't been tampered with. Upon loading, Grafana checks to see the plugin signature status.

## Badges:

> 1.- Core: this badge indicates that it has been built into Grafana.

Reviewer: ```suggestion
1.- Core: this badge indicates that the plugin has been built into Grafana.
```

<Preview>
  <PluginSignatureBadge status="internal" />
</Preview>

> 2.- Signed: its signature was successfully verified.

Reviewer: ```suggestion
2.- Signed: the plugin's signature was successfully verified.
```
